engine/src/test/java/org/destinationsol/testingUtilities/InitializationUtilities.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.testingUtilities;
+
+import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
+import org.destinationsol.SolApplication;
+import org.destinationsol.game.DebugOptions;
+import org.destinationsol.game.SolGame;
+
+public class InitializationUtilities {
+
+    public static SolGame game;
+    private static boolean initialized = false;
+
+    public static void init() {
+        if (initialized) {
+            return;
+        }
+        initialized = true;
+        DebugOptions.DEV_ROOT_PATH = "engine/src/main/resources/";
+        final LwjglApplication application = new LwjglApplication(new SolApplication());
+        try {
+            Thread.sleep(10000L);
```

Comment:
Any specific reason for this sleep of 10s? Just hoping that'll normally be enough to launch the app until it is in a good enough shape for the tests to run? In that case you might want to add a class-level javadoc snippet explaining what the class is for and why the sleep duration is what it is

Actually, in general throw javadoc on just about anything you can think of ... :D

Reply:
Javadocs in progress for other stuff too :smile: . But will add